models/battleship/game.go

```diff
@@ -6,25 +6,57 @@ import (
 	"github.com/google/uuid"
 )
 
+// const (
+// 	GameValidLowerBound = 0
+// 	GameValidUpperBound = GameGridSize - 1
+// )
+
+const (
+	GameDifficultyEasy int = iota
+	GameDifficultyNormal
+	GameDifficultyHard
+)
+
 const (
-	GameValidLowerBound = 0
-	GameValidUpperBound = GameGridSize - 1
+	GridSizeEasy   int = 6
+	GridSizeNormal int = 7
+	GridSizeHard   int = 8
 )
 
 type Game struct {
-	isFinished bool
-	Uuid       string
-	HostPlayer *Player
-	JoinPlayer *Player
-	Players    map[string]*Player
+	isFinished      bool
+	Uuid            string
+	HostPlayer      *Player
+	JoinPlayer      *Player
+	Players         map[string]*Player
+	Difficulty      int
+	GridSize        int
+	ValidLowerBound int
```

Comment:
nit: lower bound is always zero, I don't think we need to keep it in the game object. a constant would suffice.

Reply:
I agree with you...